src/app/process-page/detail/process-detail.component.ts

```diff
@@ -115,6 +127,53 @@ export class ProcessDetailComponent implements OnInit {
     );
   }
 
+  refresh() {
```

Comment:
It'd be nice if we could add specs for the new refresh / timer logic in this component.

Reply:
Added the unit tests